application/btsieve/src/load_settings.rs

```diff
@@ -1,43 +1,22 @@
 use crate::settings::Settings;
 use config::ConfigError;
-use std::path::{Path, PathBuf};
+use std::path::PathBuf;
 use structopt::StructOpt;
 
 #[derive(StructOpt, Debug)]
 pub struct Opt {
     /// Path to configuration folder
     #[structopt(short = "c", long = "config", parse(from_os_str))]
-    config_path: Option<PathBuf>,
+    config_file: PathBuf,
```

Comment:
:mega: So there is no more default location for `btsieve` now? Why not?
I think we should have the default location feature but no auto-generation.

Reply:
Sure, I re-added it.